### PR TITLE
Articles Page

### LINKED ISSUE #291: How to show non-specification publications
We have some publications that are not specifications, like press release and also various other Web pages write sometimes articles about WoT. We need ideas on how to put these in the web page.

### PR DESCRIPTION
**Pull request by egekorkan**
overtakes #302 . We should close that since it is changing the documentation page and its URL. We should not do it since it is our most popular page.

fixes #291 

More articles can be added but we can first agree on the structure

### CODE REVIEW

**Comment by egekorkan** [comment]
I have also found https://embeddedcomputing.com/technology/iot/introduction-to-web-of-things-wot-heres-everything-you-need-to-know . However, even though that it is referencing our specs, the architecture image is from the webofthings.org  and related stuff?

**Comment by egekorkan** [comment]
Call of 15.11:
- How can we pick up all the results and select from them. We should define a policy for sure. A policy should be provided with this PR.
- In this policy, we have to include a method to assess the correctness of the content of the article.
- It is better to have a shorter but correct list. In case of mistakes in articles, we can contact the authors.

**Comment by egekorkan** [comment]
I have put an initial policy document. I have thought of the different cases that can happen and it might have gotten a bit too "serious".

@koalie : Are there any guidelines we can follow for this? We want to basically include any article about WoT on our webpage and want to have a fair and transparent selection process

**Comment by ashimura** [comment]
As the WoT Team Contact, I'd suggest to @egekorkan that we have some initial chat within the group before generating a concrete policy page.

**Comment by egekorkan** [comment]
Call of 22.11:
- We should tell the visitors of the web page how to find W3C WoT articles themselves
- We can include the articles we have written (press releases and so on) or articles we have reviewed already as a group (like JSON Schema blog post).
- How to review third party articles should be discussed separately. This might be out of scope of our work in general.

**Comment by egekorkan** [comment]
Call of 20.12:
- In the search help text, it would be good to mention that searches that include "Web Thing" tend to show not standardized approaches. 
- It would be good to help people with some explanation about Web of Things before W3C activities. An FAQ would help this. Also [Wikipedia WoT](https://en.wikipedia.org/wiki/Web_of_Things#History) and our history explains it so referring to that as well would be good.

**Comment by egekorkan** [comment]
Call of 24.01: 
- The second paragraph should be more emphasized. So maybe the previous paragraph can be a light gray color or the second paragraph highlighted with a border. To be iterated next week.

File: Article_Policy.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Policy for Choosing WoT-related Articles
+
+The Web of Things Marketing Task Force collects articles about the Web of Things in their webpage at [docs/about/articles/index.html](docs/about/articles/index.html).
```
Comment (danielpeintner) [inline Article_Policy.md:3]: Not sure but shouldn't it read
**on** their webpage ?

Comment (egekorkan) [reply Article_Policy.md:3]: It should indeed be "on". Regarding "its" vs "their", I think it comes down to seeing the TF as a group of people (their) or a more abstract thingy (its). I would prefer their in this case.

Alternatively, we can just say on the official web page of W3C Web of Things

**Comment by egekorkan** [comment]
Call of 31.01: We will postpone the color change of the paragraph. This PR is ready for review by @mmccool and @ashimura

**Comment by egekorkan** [comment]
I have integrated the feedback. Netlify previews seem to have an issue so please ignore the preview link above.

**Review by relu91 — APPROVED** [review]

**Pull request merged**